src/main/java/com/codeus/winter/annotation/Autowired.java

```diff
@@ -0,0 +1,16 @@
+package com.codeus.winter.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to mark a field or constructor for dependency injection by the framework.
+ * Fields or constructors annotated with {@code @Autowired} will have their dependencies
+ * automatically injected by the framework’s dependency injection container.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD, ElementType.CONSTRUCTOR})
```

Comment:
Do we need to have an ability to inject via setter?

Reply:
Added fix to have an ability to inject via setter